tmt/convert.py

```diff
@@ -156,11 +157,11 @@ def read_nitrate(beaker_task, common_data):
         data = dict()
         echo("test case found '{0}'.".format(testcase.identifier))
         # Test identifier
-        data['tcms'] = testcase.identifier
+        data['extra-tcms'] = testcase.identifier
         # Beaker task name (taken from summary)
         if testcase.summary:
-            data['task'] = testcase.summary
-            echo(style('task: ', fg='green') + data['task'])
+            data['extra-summary'] = testcase.summary
+            echo(style('summary: ', fg='green') + data['extra-summary'])
```

Comment:
At least this should be changed to `echo(style('extra-summary: ' ...` or maybe `nitrate-summary`, otherwise there will be 'summary' twice in the output with no distinction other than that one is in Makefile section and other in Nitrate section

Reply:
Fixed in d52eb2f.